OWNERS_ALIASES

```diff
@@ -152,3 +152,10 @@ aliases:
   #
   sig-architecture-approvers: []
   sig-architecture-reviewers: []
+  
+#
+# SIG Cluster
+# Owns the cluster level application code related to Kubevirt i.e.
+# API, CRD controllers, cert-rotation, operator, etc.
+sig-cluster-approvers: []
+sig-cluster-reviewers: []
```

Comment:
Missing indent?

Reply:
After discussion on ml and with Lubo and Antonio, we agreed to stick to sig-compute for now, and remerge sig-node and sig-cluster usage in OWNER files into sig-compute again.

The split will be picked up async again.